### ISSUE TITLE
Change setup around invisible pixel

### ISSUE THREAD

**Issue by phil9001** [Leaflet/Leaflet#9363]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

I am working on a project where we are targeting a strict content security policy and we are having some issues with Leaflet. The issue is described here on a now closed issue:
https://github.com/Leaflet/Leaflet/issues/6113

In short - Leaflet creates empty pixels using 'data:', which we can not allow in our web applications with our strict CSP.

A solution was proposed in the closed issue, but it does not seem to work anymore. It would be awesome to have a solution where the invisible pixel is not set using 'data:'.

### Suggested solution

Would be great if the Leaflet package declares the empty pixel as a .gif file instead.

**Comment by AlexErmer** [comment]
I would be very interested in a solution too. Same motivation as the opener.

**Comment by mourner** [comment]
A temporary workaround would be to set `L.Util.emptyImageUrl = <...>` before a map is created. But we definitely should investigate whether WebKit still needs that empty image hack to avoid leaks.

**Comment by pierreeurope** [comment]
This should be resolved by #10068 which removed `emptyImageUrl` entirely. Could this be closed?

**Comment by mourner** [comment]
Yep!
